config/datacenters.csv

```diff
@@ -1199,8 +1212,7 @@
 84.40.0.0,84.40.63.255,Hostway Corporation,http://hostway.co.uk/
 84.53.128.0,84.53.191.255,Akamai,http://akamai.com/
 84.200.0.0,84.201.63.255,accelerated.de,http://accelerated.de/
-84.232.0.0,84.232.127.255,ServiHosting Networks S.L.,http://servihosting.es
-84.236.128.0,84.236.255.255,ServiHosting Networks S.L.,http://servihosting.es
+84.232.0.0,84.232.7.255,ServiHosting Networks S.L.,http://servihosting.es
```

Comment:
Servihosting contains a lot regional Spanish ISPs that serve residential customers. I suppose that's why you're unblocking them here?

Reply:
84.236.248.81 was reported as a false positive.  I went through much of the two blocks and found out most looked like legitimate ISPs.  I pared down the original ranges based on this
![image](https://user-images.githubusercontent.com/30205550/64465631-6274a800-d0d3-11e9-9000-9b74c236a423.png)

Reply:
Nice, thanks!